superset/assets/backendSync.json

```diff
@@ -2493,6 +2493,64 @@
       "default": "150",
       "description": "Font size for the biggest value in the list"
     },
+    "big_num_font_size": {
+      "type": "SelectControl",
+      "label": "Big Number Font Size",
+      "renderTrigger": true,
+      "clearable": false,
+      "default": 0.3,
+      "options": [
+        {
+          "label": "Tiny",
+          "value": 0.125
```

Comment:
without seeing the other PR, it's not clear what these values mean / what they translate to. for font size I'd expect units that match some sort of browser font size unit (and ideally if these are shared across multiple visualizations for header/subheaders, they would have a common meaningful interface). 

it seems a little strange that the "huge" size is not `1`? 🤔

Reply:
The values essentially mean what proportion of the component space will this header or subheader take up. I based these percentages off of the hard-coded values in the big number package. Are you wondering why we don't have a dropdown with actual values? Let me know if you prefer something else.

Reply:
got it 👍 that makes sense. could possibly leave a comment for clarification in that case.

Reply:
Done!